src/components/AdditionalInformation.jsx

```diff
@@ -20,7 +20,7 @@ const AdditionalInformation = props => {
 		subRequestTypeCity, subRequestTypeCityID, subRequestTypeZip,
 		subRequestTypeZipID } = props.formik.values;
 
-	if(props.formik.values.ContactID === null || formIncomplete(props.formik) === true){
+	if(props.formik.values.ContactID === null || formIncomplete(props.formik)){
```

Comment:
this can be

```js
!props.formik.values.ContactId
```

Reply:
No it actually cannot be or else the check never hits. We are looking for a true value to fire off this check. formIncomplete means "is the form incomplete?", !formIncomplete  would then look for the opposite value and incomplete forms would no longer trigger this check. I think at this point we need to flag re-writes and refactors in git issues and no try to tackle these at this time. I understand the code needs work and everyone is ready to jump in and pretty it up but we have a time line that is coming up quick and breaking code while trying to "fix" it at this point is counter productive.

Reply:
Check this out https://jsfiddle.net/martypowell/3fxt6kq4/

Reply:
In this case it isn't a matter of making it look pretty, it handles more than one scenario, which could potentially cause a bug. In code reviews, we are trying to make sure the changed lines are readable and avoid bugs which is absolutely not counter productive.

Reply:
When I change the code it doesn't work. When I leave it, it does. I understand what it "should" be but I'm not going to introduce errors into a working program for should be's. This is counter productive when we have a deadline of 15 days ago.